src/assets/stylesheets/preload-overlay.scss

```diff
@@ -0,0 +1,60 @@
+@import 'shared';
+
+:local(.treatment) {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+:local(.screenshot) {
+  position: absolute;
+  width: 115%;
+  height: 115%;
+  top: -40px;
+  left: -40px;
+
+  img {
+    width: 100%;
+    height: 100%;
+  }
+
+  filter: blur(10px);
```

Comment:
is it true this is expensive even if its a still image? i know we were getting burned when the underlying content was in motion since it had to re-apply the blur every frame.

Reply:
It's true that perf is much worse with moving elements, but I just did some testing and unfortunately it seems that Firefox for Android introduces ~100ms of jank on load with even a simple grayscale filter on simple page with a static background image. 
For embeds especially, it would be really nice to load the preload page as fast as possible with as little work as possible. Even more so if there are going to be multiple embeds on a page.

Reply:
Filed a bug report: https://bugzilla.mozilla.org/show_bug.cgi?id=1557908